shopelectro/tests/tests_views.py

```diff
@@ -395,3 +398,32 @@ def test_admin_autocomplete_no_results(self):
         self.assertEqual(response.status_code, 200)
         self.assertFalse(json_to_dict(response))
         self.assertNotContains(response, term)
+
+    def test_search_has_no_model_pages(self):
+        """Search page does not contain page with type=MODEL_TYPE and duplicated content."""
+        response = self.client.get(
+            f'/search/?term={self.QUOTED_SIGNLE_RESULT_TERM}',
+            follow=True,
+        )
+        result_links = BeautifulSoup(
+            response.content.decode('utf-8'), 'html.parser'
+        ).find_all(class_='search-result-link')
+        self.assertTrue(len(result_links) == 1)
+
+    def test_autocomplete_has_no_model_pages(self):
+        """Autocomplete does not contain page with type=MODEL_TYPE and duplicated content."""
+        response = self.client.get(
+            f'{reverse("autocomplete")}?term={self.QUOTED_SIGNLE_RESULT_TERM}'
+        )
+        result = list(filter(lambda d: d['type'] == 'category', json_to_dict(response)))
```

Comment:
we check, that we have pretty one page with type category.
To check, that autocomplete has no model pages, we should filter by category and compare result's size with 0.
And admin_autocomplete (test below) too